packages/ember-self-focused/addon/components/self-focused.js

```diff
@@ -1,20 +1,26 @@
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
 import layout from '../templates/components/self-focused';
 
 export default Component.extend({
   layout,
+  fastboot: service(),
   focusManager: service(),
   classNames: ['self-focused'],
 
+  isFastBoot: computed.reads('fastboot.isFastBoot'),
+
   init() {
     this._super(...arguments);
     this.get('focusManager').updateIsFirstRender();
   },
 
   didInsertElement() {
     this._super(...arguments);
-    this.get('focusManager').setNodeToBeFocused(this.element, 'insert');
+    if (!this.get('isFastBoot')) {
```

Comment:
We don't need a check here as fastboot library doesn't invoke `didInsetElement` hook in node process.

Reply:
Updated.